core/src/main/kotlin/fr/sncf/osrd/sim/DynInfra.kt

```diff
@@ -0,0 +1,10 @@
+package fr.sncf.osrd.sim
+
+import fr.sncf.osrd.new_infra.api.signaling.Signal
+import fr.sncf.osrd.new_infra.api.reservation.ReservationRoute
+import fr.sncf.osrd.new_infra.api.signaling.SignalState
```

Comment:
Those files have moved, even without conflict there will be errors when rebasing dev